templates/base.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Title Dynamic Setting:

Consider adding a comment explaining that using a template block for the title allows for dynamic setting of the page title.